### PR TITLE
rebar3 and travis.

### PR DESCRIPTION
**Pull request by cmeiklejohn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+otp_release:
+  - 18.3
+  - 19.1
+install:
+  - make
+  - ./rebar3 update
+script:
+  - make eunit
+notifications:
+  email: christopher.meiklejohn@gmail.com
```
Comment (bieniusa) [inline .travis.yml:11]: Should we send the emails to a general developer mailing list ?

File: rebar.lock [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[{<<"goldrush">>,
```
Comment (bieniusa) [inline rebar.lock:1]: Should rebar.lock be added to the repo?

**Review by bieniusa — APPROVED** [review]

File: rebar.lock (hunk 2 above)
Comment (cmeiklejohn) [reply rebar.lock:1]: Yes, it absolutely should because it makes the build reproducible.

File: .travis.yml (hunk 1 above)
Comment (cmeiklejohn) [reply .travis.yml:11]: If there is one, sure.  I put myself in here for now so it went somewhere.

**Pull request merged**